web/pages/meetings/aps/62nd_dpp_mini_conf_followups.md

```diff
@@ -0,0 +1,23 @@
+title: 62nd APS DPP - 11.04 Mini-Conference Followups
+author: Erik T. Everson
+hidetitle: True
+
+[Zoom link]: https://harvard.zoom.us/j/91600794594?pwd=L09iTGtTRUN1RmpsVnNvU05LRnNwQT09
+[chat room]: https://app.element.io/#/room/#plasmapy:openastronomy.org
+
+# 62nd APS DPP - 11.04 Mini-Conference | Post-Meeting Discussions
+### Growing an open-source software ecosystem for plasma science
+
+<div style="height: 12px"><!-- Adding vertical whitespace --></div>
+
+**Where:** On Zoom [link will be updated] <br/>
+**Day 1:** Tuesday, November 17th at 21:00 UTC / 16:00 ET / 13:00 PT <br/>
+**Day 2:** Thursday, November 19th at 19:00 UTC / 14:00 ET / 11:00 PT <br/>
+
+<div style="height: 12px"><!-- Adding vertical whitespace --></div>
+
+In order to capitalize on the incredible information shared in this year's
```

Comment:
In order to capitalize on the opportunities afforded by bringing together the diversity of speakers and attendees during this year's